packages/web-app-files/src/components/FilesList/ContextActions.vue

```diff
@@ -69,6 +73,17 @@ export default {
       required: true
     }
   },
+  setup() {
+    const activeApp = useActiveApp()
+    const isFilesAppActive = computed(() => {
+      return activeApp === FilesApp.appInfo.id
+    })
+
+    return {
+      activeApp,
+      isFilesAppActive
```

Comment:
This is not sufficient for the mixins, because they are also used inside batch actions. Although mixins allow this magic I'd prefer to be more explicit here and have `isFilesAppActive` defined in any mixin that uses it. I'd be fine for having another composable from this to make re-use easier and less verbose.

Reply:
I tried to move this to the individual mixins, but that wasn't possible as we would be mixing options and composition api. No fun.